sys-devel/gcc/gcc-10.5.0.ebuild

```diff
@@ -8,6 +8,7 @@ PATCH_GCC_VER="10.5.0"
 PATCH_VER="6"
 MUSL_VER="2"
 MUSL_GCC_VER="10.5.0"
+PYTHON_COMPAT=( python3_{10..11} )
```

Comment:
Any specific reason to skip 12 for now, or just copy/paste oversight?

Reply:
There's some regex warning I need to check out with 3.12, although it does work.